civihr_employee_portal/views/includes/civihr_employee_portal_handler_aggregated_personal_address.inc

```diff
@@ -84,4 +85,62 @@ class civihr_employee_portal_handler_aggregated_personal_address extends views_h
       filter_xss(ArrayHelper::value('country_id.name', $address))
     );
   }
+
+  /**
+   * Returns the HTML for the address as it will appear in the view.
+   *
+   * @param array $address
+   *   The address data
+   * @return string
+   *   The formatted address
+   */
+  private function formatAddressWithLabels($address) {
+    $format_street_address = '
```

Comment:
Why do you use snake case for variable names? We use camel case in almost all other parts of the code

Reply:
@mickadoo changed